chapter5.adoc

```diff
@@ -333,7 +333,7 @@ h|Field                         |Bits       |R/W    |Default    |Description
 5+h|0x0074
 h|Field                         |Bits       |R/W    |Default    |Description
 |{set:cellbgcolor:#FFFFFF}svw |15:0       |R      |DC         | Subsequent violations in the window indexed by *svi*. *svw[_j_]*=1 for the at lease one subsequent violation issued from RRID= *svi**16 + _j_.
-|{set:cellbgcolor:#FFFFFF}svi |27:16       |RW      |0         | Window's index to search subsequent violations. When read, *svi* moves forward until one subsequent violation is found or *svi* has been rounded back to the same value. After read, the window's content, *svw*, should be clean.
+|{set:cellbgcolor:#FFFFFF}svi |27:16       |WARL      |0         | Window's index to search subsequent violations. When read, *svi* moves forward until one subsequent violation is found or *svi* has been rounded back to the same value. After read, the window's content, *svw*, should be clean.

```

Comment:
> When read, *svi* moves forward until one subsequent violation is found or *svi* has been rounded back to the same value.

What if the written value is invalid.
For example, IOPMP supports 32 RRIDs.
Obviously, only first bit of `ERR_MFR.svi` is necessary.
If a programmer writes `ERR_MFR.svi=0xF` and read it back without any subsequent violations, what should the value of  `ERR_MFR.svi`  be when reading ?

Reply:
`ERR_MFR.svi` should be `0x0` or `0x1` because the IOPMP only has two windows (32 RRIDs).
However, choosing `0x0` or `0x1` is implementation-dependent.

Reply:
OK it makes sense.